### PR TITLE
fix gitlab_project_id changes every apply

### PR DESCRIPTION
**Pull request by razbensimon**
### Issue & Steps to Reproduce / Feature Request
[//]: <> (choose “resolves” “fixes” or “closes” and put link for the issue)

`gitlab_project_id` always gets changed on TF apply.

image

### Solution
- remove the `Set` call because this field doesn't exist on the blueprint
- change readme for less mistakes with personal paths
- fix broken existing test + add test

### CODE REVIEW

**Comment by razbensimon** [comment]
[Terraform Provider - GitLab Project ID](https://trello.com/c/Ng44pkjH/4165-terraform-provider-gitlab-project-id)

File: env0/resource_template.go [hunk 1]
```diff
@@ -243,7 +243,6 @@ func resourceTemplateRead(ctx context.Context, d *schema.ResourceData, meta inte
 	d.Set("description", template.Description)
 	d.Set("github_installation_id", template.GithubInstallationId)
 	d.Set("token_id", template.TokenId)
-	d.Set("gitlab_project_id", template.GitlabProjectId)
```
Comment (yaronya) [inline env0/resource_template.go:246]: Can we cover this with some test?
It feels like we aren't asserting that the value is set after create/update.

Comment (razbensimon) [reply env0/resource_template.go:246]: 3fbecee

I am not sure if I did it correctly or if I missed something on mocks. please take a look @yaronya

**Comment by razbensimon** [comment]
f9f9cd3 fixed broken test

**Comment by eranelbaz** [comment]
@razbensimon looks like something is still broken here

File: env0/resource_template.go (hunk 1 above)
Comment (razbensimon) [reply env0/resource_template.go:246]: fixed 7ae0d91

**Comment by razbensimon** [comment]
@yaronya please rereview

**Review by yaronya — APPROVED** [review]

**Pull request merged**